apis/oauth/src/main/java/org/jclouds/oauth/v2/filters/JWTBearerTokenFlow.java

```diff
@@ -50,7 +50,7 @@
 public class JWTBearerTokenFlow implements OAuthFilter {
    private static final Joiner ON_COMMA = Joiner.on(",");
 
-   private final String audience;
+   @com.google.inject.Inject(optional = true) @Named(AUDIENCE) private String audience;
```

Comment:
This is marked optional to make the injector happy when other flows don't need this, but I'd add a `checkNotNull` validation in the `filter` method so we provide a proper feedback at runtime if this value is missing.